packages/@okta/vuepress-site/docs/guides/auth-js/main/index.md

```diff
@@ -0,0 +1,302 @@
+---
+title: Auth JS fundamentals
+---
+
+<ApiLifecycle access="ie" /><br>
+<ApiLifecycle access="Limited GA" /><br>
+
+This guide explains authentication fundamentals using Auth JS, known more formally as the Okta Auth JavaScript SDK, and provides a simple SPA application to demonstrate a sign-in use case.
```

Comment:
Do we have guidance around the usage of SPA application versus SPA? 
SPA application ~ Single Page Application Application

Reply:
Thanks, good catch. We don't have guidance, but seems very reasonable to avoid using "application" after SPA (or alternatively simply spell out acronym). I'll update the doc accordingly.